syntax = "proto3";

package idvalid.rbac;

message RoleUser {
  uint64 tenant_id = 1;
  uint64 role_id = 2;
  uint64 user_id = 3;
}
